### PR TITLE
Package lo.0.1.2

### PR DESCRIPTION
**Pull request by smimram**
### `ocaml-lo.0.1.2`
Bindings for the lo library which provides functions for communicating with input controls using the OSC protocol

---
* Homepage: https://github.com/savonet/ocaml-lo
* Source repo: git+https://github.com/savonet/ocaml-lo.git
* Bug tracker: https://github.com/savonet/ocaml-lo/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

**Comment by kit-ty-kate** [comment]
The build is failing with:
```
#=== ERROR while compiling ocaml-lo.0.1.2 =====================================#
# context              2.0.4 | linux/x86_64 | ocaml-base-compiler.4.08.0 | pinned(https://github.com/savonet/ocaml-lo/releases/download/0.1.2/ocaml-lo-0.1.2.tar.gz)
# path                 ~/.opam/4.08/.opam-switch/build/ocaml-lo.0.1.2
# command              ~/.opam/4.08/.opam-switch/build/ocaml-lo.0.1.2/./bootstrap
# exit-code            127
# env-file             ~/.opam/log/ocaml-lo-15-d682c6.env
# output-file          ~/.opam/log/ocaml-lo-15-d682c6.out
### output ###
# /home/opam/.opam/4.08/.opam-switch/build/ocaml-lo.0.1.2/./bootstrap: 5: /home/opam/.opam/4.08/.opam-switch/build/ocaml-lo.0.1.2/./bootstrap: aclocal: not found
```
Also the name of the package is not meeting the recommendations. `ocaml-` prefixes should be avoided for new packages. Could you rename it to just `lo` or something?

**Comment by smimram** [comment]
Thanks, it's actually named `lo` in the current repository, I used `opam publish` for the first time and did not see that.

**Comment by smimram** [comment]
Both bugs should have been corrected now.

File: packages/lo/lo.0.1.2/opam [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    "OCAMLFLAGS=-ccopt -I/usr/local/include -cclib -L/usr/local/lib"
+    "--prefix"
+    prefix
+  ] {os = "macos"}
+  [make]
+]
+install: [
+  [make "install"]
+]
+remove: ["ocamlfind" "remove" "lo"]
```
Comment (mseri) [inline packages/lo/lo.0.1.2/opam]:
```suggestion
```

This is not necessary in opam2

File: packages/lo/lo.0.1.2/opam [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+depends: ["ocaml" "ocamlfind"]
```
Comment (mseri) [inline packages/lo/lo.0.1.2/opam]:
```suggestion
depends: ["ocaml" "ocamlfind" {build}]
```

I did not check the code, do you need ocaml find at runtime?

Comment (smimram) [reply packages/lo/lo.0.1.2/opam]: No need for ocamlfind at runtime, merging your patch.

**Comment by smimram** [comment]
All reviews from @mseri are integrated.

**Comment by mseri** [comment]
Thanks

**Comment by smimram** [comment]
Thanks to you! :)

**Pull request merged**